### PR TITLE
Add an examples section introduction.

### PR DESCRIPTION
**Pull request by avayvod**

### CODE REVIEW

**Comment by avayvod** [comment]
@mounirlamouri PTaL

File: index.html [hunk 1]
```diff
@@ -212,6 +212,51 @@
       </ul>
     </section>
     <section>
+      <h2>
+        Examples
+      </h2>
+      <p>
+        This section shows the code examples that highlight the usage of the main features of the Remote Playaback API.
```
Comment (mounirlamouri) [inline index.html]: typo: Play**a**back

File: index.html [hunk 2]
```diff
@@ -212,6 +212,51 @@
...
+        In these examples, <code>player.html</code> implements the player page controlling the remote playback and
+        <code>media.ext</code> is the media file to be played remotely. Both the page and the media are served from the
+        domain <code>http://example.org</code>. Please refer to the comments in the code examples for further details.
```
Comment (mounirlamouri) [inline index.html]: nit: https :)

File: index.html (hunk 1 above)
Comment (avayvod) [inline index.html]: Done.

File: index.html (hunk 2 above)
Comment (avayvod) [inline index.html]: Done.

File: index.html [hunk 3]
```diff
@@ -212,6 +212,51 @@
...
+      </p>
+      <section>
+        <h3>
+          Monitor availability of remote playback devices example
+        </h3>
+        <pre class="example highlight">
+&lt;!-- player.html --&gt;
+&lt;!-- the video element with custom controls that supports remote playback --&gt;
+&lt;video id="videoElement" src="http://example.org/video.ext" /&gt;
```
Comment (mounirlamouri) [inline index.html:231]: nit: https

File: index.html [hunk 4]
```diff
@@ -212,6 +212,51 @@
...
+&lt;button id="remotePlaybackBtn" style="display: none;"&gt;Play remotely&lt;/button&gt;
+&lt;script&gt;
+  // The "Play remotely" button is visible if at least one presentation display is available
```
Comment (mounirlamouri) [inline index.html:234]: nit: period at the end.

File: index.html [hunk 5]
```diff
@@ -212,6 +212,51 @@
...
+  var remotePlaybackBtn = document.getElementById("remotePlaybackBtn");
+  // show or hide the remote playback button depending on display availability
```
Comment (mounirlamouri) [inline index.html:236]: nit: "Show [...] availability."

File: index.html [hunk 6]
```diff
@@ -212,6 +212,51 @@
...
+  var handleAvailabilityChange = function(available) {
+    remotePlaybackBtn.style.display = available ? "inline" : "none";
+  };
+  var videoElem = document.getElementById("videoElement");
```
Comment (mounirlamouri) [inline index.html:240]: Alternatively, you could do:

``` js
var video = document.querySelector("video");
```

and entirely remove the id from the video element.

**Pull request merged**